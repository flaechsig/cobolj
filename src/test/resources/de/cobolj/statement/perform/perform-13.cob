IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-TMP PIC S9(2). 

PROCEDURE DIVISION.
    Perform 
    test after Varying ws-tmp from -9 by 2
    until ws-tmp is positive
        Display ws-tmp
    end-perform.

    display ws-tmp
STOP RUN.