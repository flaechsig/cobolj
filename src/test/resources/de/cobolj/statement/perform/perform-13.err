Perform mit Varying, Test After und Relational-Condition