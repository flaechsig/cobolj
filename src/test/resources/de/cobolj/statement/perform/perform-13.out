-09
-07
-05
-03
-01
+01
+01
